File: 1_Year/CursoJava/area_formas.java
import java.util.Scanner;

public class area_formas {
    public static void main(String[] args) {
        Scanner scanf = new Scanner(System.in);
        double a = scanf.nextDouble();
        double b = scanf.nextDouble();
        double c = scanf.nextDouble();
        double calc;
        scanf.close();

        calc = (a / 2) * c;
        System.out.printf("TRIANGULO: %.3f\n", calc);
    
        calc = (c * c) * 3.14159;
        System.out.printf("CIRCULO: %.3f\n", calc);
    
        calc = ((a+b) * c) / 2;
        System.out.printf("TRAPEZIO: %.3f\n", calc);
    
        calc = b*b;
        System.out.printf("QUADRADO: %.3f\n", calc);
    
        calc = a*b;
        System.out.printf("RETANGULO: %.3f\n", calc);
    }
}

File: 1_Year/AED l/www.beecrowd.com.br/Randon/area_formas.c
#include <stdio.h>

int main()
{
    double a, b, c, calc;
    scanf("%lf %lf %lf", &a, &b, &c);

    calc = (a / 2) * c;
    printf("TRIANGULO: %.3lf\n", calc);

    calc = (c * c) * 3.14159;
    printf("CIRCULO: %.3lf\n", calc);

    calc = ((a+b) * c) / 2;
    printf("TRAPEZIO: %.3lf\n", calc);

    calc = b*b;
    printf("QUADRADO: %.3lf\n", calc);

    calc = a*b;
    printf("RETANGULO: %.3lf\n", calc);

        return 0;
}